alvarocasadoc/day1/day1.exs

```diff
@@ -0,0 +1,22 @@
+# AdventOfCode2018
+# @alvarocasadoc
+# Day 1
+
+defmodule Day1 do
+
+  # Part 1
+  def sum_frequencies(input) do
+    items = Enum.map(input, fn x -> Integer.parse(x) |> elem(0) end)
+    Enum.sum(items)
```

Comment:
Okay, so if I understand correctly the functions for collections are inside the module `Enum`?

Then, as Integer.parse returns a Tuple, the `|>` thing is `then`?

If that's so you could do which is still readable and keeps the function more `mathematical` (but that's personal preference)

```elixir
def sum_frequencies(input) do
  Enum.map(input, fn x -> Integer.parse(x) |> elem(0) end) |> Enum.sum()
end
```

Good job 👍 I really like this

Reply:
Yes, you're right! `|>` works as a pipe operator.